src/powfacpy/base_interface.py

```diff
@@ -401,29 +400,23 @@ def delete_obj(self, obj_or_path, condition=None, parent_folder=None, error_if_n
       error_if_non_existent=error_if_non_existent,
       include_subfolders=include_subfolders)
     for o in obj:
-      success = o.Delete()
-      """
-      if not success == 0:
-        o.Deactivate()
-        o.Delete()
-      # Due to a PF bug, study cases need special treatment.
-      # Here, Delete() returns 0 even if it was not successful.  
-      elif o.GetClassName() == "IntCase":
-        try:
-          o.Deactivate()
-          o.Delete()
-        except:
-          pass  
-      """
+      o.Delete()
       # 'IsDeleted' seems to be the savest way to check whether an object has been deleted. 
       if not o.IsDeleted():
-        try:
-          o.Deactivate()
-          o.Delete()
-        except:
-          raise ArgumentError(r"Object {o} cannot be deleted.")
+        if not o.IsDeleted(): 
```

Comment:
merge conflicts with last pull request -- resolve manually, as discussed.

Reply:
Not relevant with latest commit.